myhdl/conversion/_toVHDL.py

```diff
@@ -782,6 +782,15 @@ def visit_BoolOp(self, node):
         self.write(")")
 
     def visit_UnaryOp(self, node):
+        if isinstance(node.op, ast.USub):
+            if isinstance(node.operand, ast.Num):
+                n = node.operand.n
+                newnode = copy(node.operand)
+                newnode.n = -n
+                newnode.vhd = node.vhd
+                newnode.vhdOri = node.vhdOri
+                self.visit(newnode)
+                return
```

Comment:
@jmgc This change seems to fix test_numass.py for VHDL.
Could you explain it to me?